wire-runtime-swift/src/test/swift/sample/Dinosaur.swift

```diff
@@ -12,17 +12,27 @@ public struct Dinosaur {
     /**
      * URLs with images of this dinosaur.
      */
-    public var picture_urls: [String]
+    public var picture_urls: [String] = []
     public var length_meters: Double?
     public var mass_kilograms: Double?
     public var period: Period?
     public var unknownFields: Foundation.Data = .init()
 
+    public init(configure: (inout Self) -> Void = { _ in }) {
+        configure(&self)
+    }
+
+}
+
+#if WIRE_INCLUDE_MEMBERWISE_INITIALIZER
+extension Dinosaur {
+
+    @_disfavoredOverload
```

Comment:
Fancy!

Reply:
hehe I realized I would need it immediately for a bunch of types 🙂